modin/numpy/arr.py

```diff
@@ -397,10 +397,30 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
                 if isinstance(input, pd.Series):
                     input = input._query_compiler.to_numpy().flatten()
                 args += [input]
+            out_kwarg = kwargs.get("out", None)
+            if out_kwarg is not None:
+                # If `out` is a modin.numpy.array, `kwargs.get("out")` returns a 1-tuple
+                # whose only element is that array, so we need to unwrap it from the tuple.
+                out_kwarg = out_kwarg[0]
+            kwargs.pop("out", None)
+            where_kwarg = kwargs.get("where", None)
+            if where_kwarg is not None:
+                if isinstance(where_kwarg, type(self)):
+                    where_kwarg = where_kwarg._to_numpy()
+                kwargs["where"] = where_kwarg
```

Comment:
```suggestion
                if isinstance(where_kwarg, type(self)):
                    kwargs["where"] = where_kwarg._to_numpy()
```
no point of re-assigning stuff which didn't change 😺 

also, why do we want to cast `where` to plain numpy?